docs/projects/ActivityDekho/Activity-Dekho-DirectoryStructure.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Activity Dekho
+
+AICTE Dairy - To Help Organize and Maintain your Points
+
+Visit Activity-Dekho(https://github.com/tcet-opensource/Activity-Dekho)
+
+The following is the TCET's Activity-Dekho FOLDER STRUCTURE.
+
+
+``` bash
+
+├── public
+│   └── All static files
+│   
+└── src
+    ├── components
+    │   └── All reusable and abstracted components are here
+    │   
+    ├── firebase
+    │   └── All firebase config is here and firebase queries are inside firestore folder inside here
+    │   
+    ├── machines
+    │   └── xstate Finite State Machines which hold all the logic exist here
+    │   
+    ├── pages
+    │   └── all files here are a page
+    │  
+    └── utils
```

Comment:
utils -> enums.ts //contains all constant grouped types
utils -> types.ts //contains all typescript type definations
utils -> utils.ts //contains all dynamic reusable scripts

Reply:
Changes are done. Checkout the updated file.